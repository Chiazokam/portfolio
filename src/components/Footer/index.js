import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import Button from '@material-ui/core/Button';
import Container from '@material-ui/core/Container';
import logo from '../../images/logo.svg';

const Footer = () => {

  return (
    <div className='footer'>
        <Container fixed>
            <div className='footer-text'>
                <p>Designed by </p> &nbsp; &nbsp; <img src={logo} alt='chiazokam' />
            </div>
        </Container>
    </div>
  );
}

export default Footer;
