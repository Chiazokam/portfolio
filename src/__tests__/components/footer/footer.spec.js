import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../../components/Footer';

describe('Footer component', () => {
    it('renders successfully', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
    })
});
