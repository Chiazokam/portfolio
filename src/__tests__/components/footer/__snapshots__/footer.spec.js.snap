// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders successfully 1`] = `ShallowWrapper {}`;
